### DISCUSSION TITLE
Types from @mantine/core don't work with autocompletion

### DISCUSSION THREAD

**Discussion by itmecho** [pnpm/pnpm#3708] [Q&A]
Hi! I'm trying to use `pnpm` on a new project with next and mantine but the autocompletion in both neovim and vs code doesn't work for `@mantine/core` when installed with `pnpm`.

Installing with `npm` works fine. Interestingly, the autocomplete for `@mantine/hooks` works fine with `pnpm`.

I'm a bit lost on where the actual issue is so I'm hoping someone from either the `mantine` project, `pnpm` project or `typescript-language-server` can help!

Replication steps are in the thread on the mantine discussion

Mantine discussion: https://github.com/mantinedev/mantine/discussions/224
Typescript Language Server issue: https://github.com/typescript-language-server/typescript-language-server/issues/254

**Comment by marcesengel** [comment]
I'm facing this issue with various other packages as well, might have to go back to using `npm`...
